Model/ConfigProvider.php

```diff
@@ -84,6 +82,11 @@ protected function getAllPaymentMethods()
 
         $response = $this->apidata->getResponse($uri, '', $merchantId, $merchantSecret, $method);
 
+        $status = $response['status'];
+        if ($status === 422 || $status === 400 || $status === 404 || $status === 401 || !isset($status)){
```

Comment:
I would change this to something like 
$status = $isset($response['status'])?$response['status']:false;
if ($status === 422 || $status === 400 || $status === 404 || $status === 401 || $status === false){
since it can cause error, but letting up to you.